### PR TITLE
First draft of "Types of Data Visualizations" Section

### LINKED ISSUE #41: Write tutorial: Types of data visualizations
Topic: Types of data visualizations

Topics covered (with matplotlib)

1. Barplots
2. Scatter plots
3. Density plots
4. Box and whisker plot
5. Heatmap

Starter file: https://github.com/ploomber/sql/blob/main/course-material/visualizing-your-sql-queries/types-of-visualizations.md

### PR DESCRIPTION
**Pull request by jpjon**
## Issue number
#41
Closes #41 

## Checklist before requesting a review

- [ ] Performed a self-review of my code
- [ ] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)

----
:books: Documentation preview :books:: https://ploomber-sql--48.org.readthedocs.build/en/48/

### CODE REVIEW

**Comment by lfunderburk** [comment]
Hi @jpjon 

Please fix existing conflicts due to the change in structure in main

Moving this from blocked to doing

**Comment by jpjon** [comment]
Main branch has been merged and is ready for review.

**Review by TegveerG — DISMISSED** [review]
Looks good to me, nice work!

**Review by lfunderburk — CHANGES_REQUESTED** [review]
Hi @jpjon 

It might be confusing for the audience to start creating datasets from scratch. 

It will also cause confusion since the seaborn and plotly tutorials are using the 2 schemas we've been working with. 

Also noticing that half of this tutorial revolves around the creation of the data. It distracts a bit from the main topic. 

Thinking back on discussion, I think it makes sense to stick with one of the tables in the datasets to keep this intro simple.

To keep things easy, maybe you can reuse some of the queries from the getting started section.

**Comment by jpjon** [comment]
The section has been revised based on Laura's suggestions. Should be ready for review!

File: docs/visualizing-your-sql-queries/types-of-visualizations.md [hunk 1]
```diff
@@ -1,14 +1,218 @@
...
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
 ---
 
 # Types of data visualizations
 
+Welcome to the <b>Visualizing your SQL queries</b> module of the course! This part of the course will introduce data visualizations and commonly used packages. After getting familiar with the types of data visualizations and visualization packages, we'll revisit SQL and teach you JupySQL's unique feature of utilizing `ggplot` to visualize queries.
 
+This module will teach you the `seaborn` and `plotly` packages. Before we get into the details of each package, we first introduce the common types of data visualization with one of the most basic visualization packages: `matplotlib`. The purpose of this section is to not teach you the ins and outs of `matplotlib`, but more so to introduce some basic data visualizations.
```
Comment (neelasha23) [inline docs/visualizing-your-sql-queries/types-of-visualizations.md:18]: `In this module we will learn about...`

File: docs/visualizing-your-sql-queries/types-of-visualizations.md [hunk 2]
```diff
@@ -1,14 +1,218 @@
...
+
+plt.xlabel("Job")
+plt.ylabel("Count")
+plt.xticks(rotation=45)
+plt.title("Count of Each Job")
+
+plt.show()
+```
+
+The second in the above code cell is really all we need to create a baseline bar plot. The remaining lines are supplemental elements that labels and customizes the y-axis, x-axis, size, and title of the plot.
```
Comment (neelasha23) [inline docs/visualizing-your-sql-queries/types-of-visualizations.md:121]: second what?

File: docs/visualizing-your-sql-queries/types-of-visualizations.md [hunk 3]
```diff
@@ -1,14 +1,218 @@
...
+plt.ylabel("Count")
+plt.xticks(rotation=45)
+plt.title("Count of Each Job")
+
+plt.show()
+```
+
+The second in the above code cell is really all we need to create a baseline bar plot. The remaining lines are supplemental elements that labels and customizes the y-axis, x-axis, size, and title of the plot. 
+
+We can easily see that "management" and "blue-collar" jobs are the most prominent job category in this data set. Box plots are a great option when you need to visualize distributions of groups in a categorical variable.
```
Comment (neelasha23) [inline docs/visualizing-your-sql-queries/types-of-visualizations.md:123]: Please highlight management and blue-collar jobs using ``

**Comment by jpjon** [comment]
Thank you for the suggestions @neelasha23. They have been implemented and the PR is ready for review.

**Review by neelasha23 — APPROVED** [review]
lg!
Please tick the checklist boxes when opening a PR

**Pull request merged**